content/articles/setting-up-kali-linux-for-dual-boot/index.md

```diff
@@ -0,0 +1,124 @@
+- Enter and confirm the `password` entry.
+
+> **Note:** Remember your `username` and `password` since they are case sensitive and you will require them on every login session.
+
+### Creating disk partitions
+Since we have already created space for Kali Linux, we will do this manually. 
+- Select the `manual` option - This will display a list of all the drives on the PC.
+- Choose the drive with `free space` - This is the space we created. We will further create two partitions from the available free space.
+
+One of the partitions will be the `root partition` for Kali Linux and the other will be the `swap area`.
+
+First, let's create a **root partition** - This is where the operating system will be installed.
+
+Choose the free space option and select the options as shown below.
+
+- Select **create a new partition option**.
+- Allocate a space **95% or more** of the free space to the root partition.
+- Select the location of the new partition as the **beginning** of the drive.
+- Choose the File system as **Ext4 journaling**.
+- Mounting Point **/** - the root file system.
+
+**Note:** Click `done setting up the partition` before proceeding. Otherwise, you will have to start the partitioning process over again.
+
+### Creating a swap partition
+Swap area access virtual RAM if the system has less amount of physical memory. If your machine has 16GB of RAM or more, there is no need to create a swap partition.
+
+- Select the options same as creating a root partition.
+- Select the `swap area` option as the use of the partition.
+- Remember to select `done setting up the partition`.
+
+At this point, you have successfully created a partition.
+Click `Finish partitioning and write changes to disk` then hit the continue button.
+
+On the **write changes to disk** option, select `Yes` and wait for the installer set-up to `install the base system`.
+
+### Software selection
+Here you have to choose your favorite desktop environment. In my case, I'll' choose the `Xfce (Kali default desktop environment)`.
+
+Lastly, click continue without altering the other changes. Wait for the installation process to complete. This might take a while.
+
+When the installation process is complete, reboot your machine and remove the pen drive.
+
+### What happens after rebooting?
+After rebooting, Kali Linux will load into `GRUB boot loader`. This is where you're given the option to either boot into Kali Linux or Windows. Select Kali Linux in this case.
+
+The login screen will be displayed as shown below:
+![Login page](/engineering-education/setting-up-kali-linux-for-dual-boot/login.png)
+
+Enter your login credentials, that is, your `username` and `password` to proceed.
+
+Upon successful login, you will be able to access Kali Linux home screen on your machine. If it is your first time using Kali Linux, you can go through [this article](https://www.section.io/engineering-education/getting-started-with-kali-linux/) for more information.
+
```

Comment:
Just before the summary, give general importance of installing a Linux-based OS alongside Windows.